src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -41,10 +44,57 @@ public void Initialize(ExtensionConfigContext context)
             {
                 throw new ArgumentNullException("context");
             }
+
+            INameResolver nameResolver = context.Config.NameResolver;
+            IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
```

Comment:
A helper method is great. But having it assign to itself is wrong, particularly when we're getting converterManager from the config. 

---
In reply to: [63605331](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63605331) [](ancestors = 63605331)